.github/workflows/maven.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       uses: GabrielBB/xvfb-action@v1
       with:
        run: >- 
-        mvn -V -B -D maven.test.failure.ignore=true clean verify
+        mvn -V -B -fae clean verify
     - name: Upload Test Results for Java-${{ matrix.java }}
       uses: actions/upload-artifact@v3
```

Comment:
The tests results are currently only uploaded, if all tests succeed. So I suggest configuring the upload job so that the test results are always transferred back to GH.
```
    - name: Upload Test Results for Java-${{ matrix.java }}
      if: always()
      uses: actions/upload-artifact@v3
```

Reply:
shouldn't be be:
```
    - name: Upload Test Results for Java-${{ matrix.java }}
      uses: actions/upload-artifact@v3
      if: always()
```

PR updated accordingly. If you order is correct, I will adjust the PR again

Reply:
I don't think the order is important, so that one should also work.